jina/helper.py

```diff
@@ -1684,3 +1684,41 @@ def _telemetry():
             pass
 
     threading.Thread(target=_telemetry, daemon=True).start()
+
+
+_DEFAULT_GRPC_OPTION = {
+    'grpc.max_send_message_length': -1,
+    'grpc.max_receive_message_length': -1,
+    # for the following see this blog post for the choice of default value https://cs.mcgill.ca/~mxia3/2019/02/23/Using-gRPC-in-Production/
+    'grpc.keepalive_time_ms': 10000,
+    # send keepalive ping every 10 second, default is 2 hours.
+    'grpc.keepalive_timeout_ms': 5000,
+    # keepalive ping time out after 5 seconds, default is 20 seconds
+    'grpc.keepalive_permit_without_calls': True,
+    # allow keepalive pings when there's no gRPC calls
+    'grpc.http2.max_pings_without_data': 0,
+    # allow unlimited amount of keepalive pings without data
+    'grpc.http2.min_time_between_pings_ms': 10000,
+    # allow grpc pings from client every 10 seconds
+    'grpc.http2.min_ping_interval_without_data_ms': 5000,
+    # allow grpc pings from client without data every 5 seconds
+}
+
+
+def _get_grpc_server_options(option_from_args: Dict) -> List[Tuple[str, Any]]:
+    """transform dict of args into grpc option, will merge the args wit the default args
+    :param option_from_args: a dict of argument
+    :return: grpc option i.e a list of tuple of key value
+    """
+
+    option_from_args = (
+        {**_DEFAULT_GRPC_OPTION, **option_from_args}
+        if option_from_args
+        else _DEFAULT_GRPC_OPTION
+    )  # merge new and default args
+
+    options_from_dict = []
+    for key, value in option_from_args.items():
+        options_from_dict.append((key, value))
```

Comment:
can't we just do `return list(option_from_args.items())`?

Reply:
done